src/views/commentsViews/commentsCard.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  Card,
+  CardText,
+  CardHeader,
+  UncontrolledCollapse,
+  DropdownItem,
+  DropdownMenu,
+  DropdownToggle,
+  UncontrolledDropdown,
+  Button
+} from "reactstrap";
+import InsertComment from "../../components/comments/newComment";
+import "../../assets/comment/comment.scss";
+import PropTypes from "prop-types";
+import Moment from "react-moment";
+
+const CommentCard = props => (
+  <div className="comment">
+    <ViewCard {...props} />
+    <Button outline color="primary" id={`reply${props.id}`}>
+      Replies
+    </Button>
+    <UncontrolledCollapse toggler={`#reply${props.id}`} className="replies">
+      <InsertComment value={"Reply"} holder={"Enter your reply here..."} id={props.id}/>
+      {props.replies.map(item => (
+        <ViewCard {...item} key={item.id} />
+      ))}
+    </UncontrolledCollapse>
+  </div>
+);
+const ViewCard = props => (
+  <React.Fragment>
```

Comment:
Change this `React.Fragment` to just `Fragment` by destructuring it the way you would for `Component`